demo/repo/Jenkinsfile

```diff
@@ -50,3 +58,11 @@ def runTests(servers, duration) {
         junit '**/target/surefire-reports/TEST-*.xml'
     }
 }
+
+def milestone(n) {
+    try {
+        steps.milestone(n)
+    } catch (NoSuchMethodError _) {
```

Comment:
If that were any dirtier it would be banned as an obscenity... but it gets the job done and is better than the alternative.